spec/features/roles_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe 'assigning roles' do
+  context 'with admin privilege' do
+    it 'assigns one role to user' do
+      admin = create :user, admin: true
+      login_as admin
```

Comment:
Also when I use sign_in I get a undefined method error in comparision to using login_as.